### PR TITLE
New package: rott-1.1.2

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

**Comment by diogoleal** [comment]
@Duncaen 
added ```build_wrksrc=rott```

File: srcpkgs/rott/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'rott'
+pkgname=rott
+version=1.1.2
+revision=1
+only_for_archs="i686 x86_64"
```
Comment (xtraeme) [inline srcpkgs/rott/template]: why?

Comment (diogoleal) [inline srcpkgs/rott/template]: with armv7-musl: 
```
armv7l-linux-musleabihf-gcc -specs=/void-packages/common/environment/configure/gccspecs/hardened-cc1  -fstack-protector-strong -D_FORTIFY_SOURCE=2  -O2 -pipe -march=armv7-a -mfpu=vfpv3 -mfloat-abi=hard   -I/usr/armv7l-linux-musleabihf/usr/include -Wall -Wno-unused -I/usr/armv7l-linux-musleabihf/usr/include/SDL -D_GNU_SOURCE=1 -D_REENTRANT      -DSHAREWARE=1 -DSUPERROTT=0 -DSITELICENSE=0 -DUSE_SDL=1 -DPLATFORM_UNIX=1   -c -o dosutil.o dosutil.c
dosutil.c:201:22: fatal error: execinfo.h: No such file or directory
 #include <execinfo.h>
                      ^
compilation terminated.
<builtin>: recipe for target 'dosutil.o' failed
make: *** [dosutil.o] Error 1
```

Comment (xtraeme) [inline srcpkgs/rott/template]: This is not a good reason... you can make a patch that enables execinfo/backtrace only if `__GLIBC__` is defined.

**Comment by Vaelatern** [comment]
Is work on this stale? Will not be accepted as is due to spurious `only_for_archs`.

**Pull request closed without merging**